### PR TITLE
Set all string declarations using f-strings

### PR DESCRIPTION
**Pull request by amylily1011**
Resolves: #141

### CODE REVIEW

**Comment by lukehinds** [comment]
hi @amylily1011 

thanks for your work, we have a few things to fix up, but surprisingly a small amount for such a voluminous change: https://travis-ci.org/keylime/keylime/builds/584610418?utm_source=github_status&utm_medium=notification

I can go over this with you on Monday!

**Comment by amylily1011** [comment]
Sure @lukehinds !
I'll be online around 4pm BKK time.

**Comment by amylily1011** [comment]
I got one error in the test, but it wasn't relevant to the fstring. So I pushed the new commit for now, so you can have a look at it :)

**Comment by lukehinds** [comment]
Thanks amy, will try to take a look tomorrow.

**Comment by lukehinds** [comment]
Restarting CI Build to see if failures still present.

File: keylime/ca_util.py [hunk 1]
```diff
@@ -56,8 +56,8 @@
 elif common.CA_IMPL=='openssl':
     from keylime import ca_impl_openssl as ca_impl
 else:
-    raise Exception("Unknown CA implementation: %s"%common.CA_IMPL)
-from M2Crypto import X509, EVP, BIO
+    raise Exception(f'Unknown CA implementation: {common.CA_IMPL}')
+    from M2Crypto import X509, EVP, BIO
```
Comment (lukehinds) [inline keylime/ca_util.py]: Found the issue making CI Fail.

Line 60 is indented which means the import does not happen (it only happens as part of the exception occurs..."raise Exception"

Just bring the 4 from back 4 spaces and its all good.

Comment (amylily1011) [reply keylime/ca_util.py]: Looking at it right now :)

**Comment by lukehinds** [comment]
Let me know if you need some help with the conflicting files.

**Comment by amylily1011** [comment]
I think I found something I might have missed.

**Pull request closed without merging**